src/state/notifications/sagas.ts

```diff
@@ -22,4 +42,31 @@ export function* createNotificationEmailSaga(action: CreateNotificationEmailActi
   }
 }
 
-export default [takeEvery(NotificationAction.CreateNotificationEmail, createNotificationEmailSaga)];
+export function* setNotificationEmailSaga(action: SetNotificationEmailAction | unknown) {
+  const { meta, payload } = action as SetNotificationEmailAction;
+  const email = payload.email;
+  try {
+    yield StoreService.setStoreValue('email', email);
```

Comment:
I would rather keep notifications in redux than directly in phone storage.

Reply:
I think we accepted this time ago we will keep also in storage :)